src/qml/qmlchainpresetmodel.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 #include <QAbstractListModel>
-#include <QtQml>
+#include <QQmlEngine>
```

Comment:
This is also out of scope of this PR.

Reply:
This is necessary, that Clazy passes, should I move it to an own PR?

Reply:
It is not very important. I actually prefere to have everything fixed either in one go or before doing the switch. This as well as the nan-infinity-disabled issue. We need to do it anyway at one point, let's avoid intermediate workarounds.